src/status_im/ui/screens/syncing/sheets/enter_password/styles.cljs

```diff
@@ -0,0 +1,9 @@
+(ns status-im.ui.screens.syncing.sheets.enter-password.styles)
+
+(def header-text
+  {:font-size 27
+   :font-weight :600
```

Comment:
why are we not using quo 2 for font pieces?

Reply:
hmm this was made 3 months ago, the idea was to swiftly get the UI up and running, I shall refactor :)

Reply:
fixed in latest commit :)